test/system/070-build.bats

```diff
@@ -32,9 +31,8 @@ EOF
 
 # Regression from v1.5.0. This test passes fine in v1.5.0, fails in 1.6
 @test "podman build - cache (#3920)" {
-    skip_if_remote "FIXME: pending #7136"
-    if is_remote && is_rootless; then
-        skip "unreliable with podman-remote and rootless; #2972"
+    if is_rootless; then
```

Comment:
I don't remember the reason for this one, but if I read the logic correctly, and if you've really fixed this for remote (safe bet, since the original skip was varlink) -- shouldn't you just remove the `if/skip` entirely? As it was, it would run rootless-local, and now it won't.